invenio_app_rdm/theme/assets/semantic-ui/js/invenio_app_rdm/utils.js

```diff
@@ -4,26 +4,36 @@
 // Invenio RDM Records is free software; you can redistribute it and/or modify it
 // under the terms of the MIT License; see LICENSE file for more details.
 
+import axios from "axios";
+
 /**
  * Wrap a promise to be cancellable and avoid potential memory leaks
  * https://reactjs.org/blog/2015/12/16/ismounted-antipattern.html
  * @param promise the promise to wrap
  * @returns {Object} an object containing the promise to resolve and a `cancel` fn to reject the promise
  */
- export const withCancel = (promise) => {
-    let isCancelled = false;
-  
-    const wrappedPromise = new Promise((resolve, reject) => {
-      promise.then(
-        (val) => (isCancelled ? reject('UNMOUNTED') : resolve(val)),
-        (error) => (isCancelled ? reject('UNMOUNTED') : reject(error))
-      );
-    });
-  
-    return {
-      promise: wrappedPromise,
-      cancel() {
-        isCancelled = true;
-      },
-    };
+export const withCancel = (promise) => {
+  let isCancelled = false;
+
+  const wrappedPromise = new Promise((resolve, reject) => {
+    promise.then(
+      (val) => (isCancelled ? reject("UNMOUNTED") : resolve(val)),
+      (error) => (isCancelled ? reject("UNMOUNTED") : reject(error))
+    );
+  });
+
+  return {
+    promise: wrappedPromise,
+    cancel() {
+      isCancelled = true;
+    },
   };
+};
+
+const apiConfig = {
+  withCredentials: true,
+  xsrfCookieName: "csrftoken",
+  xsrfHeaderName: "HTTP_X_CSRFTOKEN",
```

Comment:
I'm not entirely sure what's to be done:
a) Change it in `Invenio-REST` and therefore here (We need to coordinate with ILS people since they use this variable everywhere)
b) Leave it to be solved along with the invenio-rest issue

Reply:
In ``config.py`` in App-RDM add 

```python
CSRF_HEADER = "X-CSRFToken"
```

Replace ``HTTP_X_CSRFTOKEN`` with ``X-CSRFToken`` in JS code (in App-RDM and React-Invenio-Deposit)